spec/factories/payroll_runs.rb

```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
   factory :payroll_run do
-    created_by { "123" }
+    created_by { build(:dfe_signin_user) }
```

Comment:
Should this be explicitly be defined as an association? (just looking at Tekin's comment here: https://github.com/DFE-Digital/dfe-teachers-payment-service/pull/731#discussion_r359816862)

Reply:
Good catch. Done! 👍